templates/targets/x86_64/generic/flake.nix

```diff
@@ -55,10 +76,33 @@
               # To handle the majority of laptops we need a little something extra
               # TODO:: SEE: https://github.com/NixOS/nixos-hardware/blob/master/flake.nix
               # nixos-hardware.nixosModules.lenovo-thinkpad-x1-10th-gen
+
+              # Required for successful installation via nixos-anywhere
+              boot.loader.grub = {
```

Comment:
```
boot = {
      loader.systemd-boot.enable = true;
      loader.efi.canTouchEfiVariables = true;
      loader.efi.efiSysMountPoint = "/boot/efi";
    };

```
Would this not be more appropriate for a laptop/desktop device?

Reply:
Sure, I'll retest it with systemd-boot